sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java

```diff
@@ -443,6 +440,28 @@ protected void validatePositive(final Double dValue) {
     }
 
     /**
+     * Validates if a parameter is in a certain range, throws an exception describing the error if not
+     * @param dValue value to be checked
+     * @param minimum lower bound, NaN if no lower bound
```

Comment:
use Double.NegativeInfinity for lower bound and Double.PositiveInfinity for upper bound, throw if it is NaN.  That way it makes sense

Reply:
update the comments